docs/02-app/01-building-your-application/09-deploying/01-production-checklist.mdx

```diff
@@ -0,0 +1,108 @@
+
+### Type safety
+
+- **TypeScript and [TS Plugin](/docs/app/building-your-application/configuring/typescript):** Use TypeScript and the TypeScript plugin for better type-safety, and to help you catch errors early.
+
+## Before going to production
+
+Before going to production, you can run `next build` to build your application locally and catch any build errors, then run `next start` to measure the performance of your application in a production-like environment.
+
+### Core Web Vitals
+
+- **[Lighthouse](https://developers.google.com/web/tools/lighthouse):** Run lighthouse in incognito to gain a better understanding of how your users will experience your site, and to identify areas for improvement.
+- **[`useReportWebVitals` hook](/docs/app/api-reference/functions/use-report-web-vitals):** Use this hook to send [Core Web Vitals](https://web.dev/articles/vitals) data to analytics tools.
+
+### Regions and runtimes
+
+- **[Regions](https://vercel.com/docs/edge-network/regions):** Ensure your application and database are deployed in the same region to reduce [latency](https://developer.mozilla.org/en-US/docs/Web/Performance/Understanding_latency).
+- **[Edge Runtime](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes):** If using the Edge Runtime, consider that multiple requests from the Edge to your database might take longer than rendering on the Node.js runtime which can be closer to your database. Consider pinning your Edge location to a [preferred region](/docs/app/api-reference/file-conventions/route-segment-config#preferredregion) to reduce latency.
+
+### Analyzing bundles
+
+Use the [`@next/bundle-analyzer` plugin](/docs/app/building-your-application/optimizing/bundle-analyzer) to analyze the size of your JavaScript bundles and identify large modules and dependencies that might be impacting your application's performance.
+
+Additionally, the following tools can you understand the impact of adding new dependencies to your application:
+
+- [Import Cost](https://marketplace.visualstudio.com/items?itemName=wix.vscode-import-cost)
+- [Package Phobia](https://packagephobia.com/)
+- [Bundle Phobia](https://bundlephobia.com/)
+- [bundlejs](https://bundlejs.com/)
+
+## After deployment
+
+Depending on where you deploy your application, you might have access to additional tools and integrations to help you monitor and improve your application's performance.
+
+For Vercel deployments, we recommend the following:
+
+- **[Logging](https://vercel.com/integrations#logging), [Monitoring](https://vercel.com/integrations#monitoring), and [Observability](https://vercel.com/integrations#observability):** See Vercel's integrations page for a list of available tools and services to help you monitor your application's performance.
```

Comment:
We might want to link to the Vercel logs/monitoring products, and then say "or use integrations". Also, let's add the UTM params for Vercel links 👍

Reply:
Thank you, will do 🙏🏼